sample-app/src/components/DropdownSection.tsx

```diff
@@ -67,11 +115,17 @@ export default function DropdownSection({
     }
 
     if (evt.key === 'ArrowDown' || evt.key === 'ArrowRight') {
-      incrementOptionFocus();
+      updateToNextFocusElement();
     } else if (evt.key === 'ArrowUp' || evt.key === 'ArrowLeft') {
-      decrementOptionFocus();
+      updateToPrevFocusElement();
     } else if (evt.key === 'Enter') {
       onSelectOption(options[focusedOptionIndex].value, focusedOptionIndex);
+      if (focusedLinkIndex !== -1) {
+        const verticalLinks = options[focusedOptionIndex]?.verticalLinks;
+        if (verticalLinks) {
+          browserHistory.push(`/${verticalLinks[focusedLinkIndex].verticalKey}`);
```

Comment:
Can you explain why we're modifying browserHistory here?

Reply:
it's from useHistory hook in React Router and we need to modify it like so to navigate to a different page programmatically in the router.

Reply:
When I prototyped this I used a 'Link' from 'react-router-dom'. Could that work instead? (See: https://github.com/yext-pages/verizon-demo/pull/37/files)

Reply:
A Link would only work when click on (and then and routes to whatever 'to' field points to), but to navigate programmatically, like when user hit enter on the link option, I believe I tried that before and it didn't work, so I ended up using the router history